test/kubernetes-test.sh

```diff
@@ -31,16 +31,21 @@ helm template --validate install/kubernetes/cilium \
 
 kubectl apply -f cilium.yaml
 
-while true; do
-    result=$(kubectl -n kube-system get pods -l k8s-app=cilium | grep "Running" -c)
-    echo "Running pods ${result}"
-    if [ "${result}" == "2" ]; then
+pollCiliumPods () {
+    while true; do
+        result=$(kubectl -n kube-system get pods -l k8s-app=cilium | grep "Running" -c)
+        echo "Running pods ${result}"
+        if [ "${result}" == "2" ]; then
 
-        echo "result match, continue with kubernetes"
-        break
-    fi
-    sleep 1
-done
+            echo "result match, continue with kubernetes"
+            break
+        fi
+        sleep 1
+    done
+}
+
+export -f pollCiliumPods
```

Comment:
Why is this line necessary?

Reply:
That's because we are trying to create a subprocess as I am using timeout which is a Linux command. So, I just exported my function so that it could be used by child process.

Reply:
:+1: , then let's add an unexport of it after its use.